mandatory/1-fix-functions.js

```diff
@@ -21,30 +19,30 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = 10;
```

Comment:
Your solution is not wrong, but the variable name is called `isBigEnough` so you should do the test if the number is bigger than 10 here :)